client/my-sites/checkout/checkout-thank-you/marketplace/style.scss

```
@@ -15,17 +18,25 @@
 	.thank-you__container-header {
 		min-height: auto;
 		padding: 0;
-		padding-top: 10px;
+
+		margin-top: 30px;
+
+		@include break-mobile {
+			margin-top: 60px;
+		}
 	}
 
 	.thank-you__body {
-		margin: 50px 0;
-
 		> div {
-			width: auto;
+
 			display: flex;
 			flex-direction: column;
-			align-items: center;
+			width: 100%;
+			align-items: left;
```

```suggestion
			align-items: start;
```

left isn't a valid property, so maybe this is what you meant?

---

More likely not needed at all!

```suggestion
```